src/config.rs

```diff
@@ -79,6 +80,14 @@ impl Default for TermvizConfig {
                     g: 200,
                 },
             }],
+            image_topics: vec![ListenerConfig {
```

Comment:
Same here maybe move default to the implementation

Reply:
this is just for the example config, or what do you mean? the config itself should be the next big topic to be worked on, prs welcome :dancers: